src/Selenium2Library/keywords/_selectelement.py

```diff
@@ -8,15 +8,24 @@ class _SelectElementKeywords(KeywordGroup):
 
     # Public
 
-    def get_list_items(self, locator):
-        """Returns the values in the select list identified by `locator`.
+    def get_list_items(self, locator, label=True):
+        """Returns the labels or values in the select list identified by `locator`.
 
         Select list keywords work on both lists and combo boxes. Key attributes for
         select lists are `id` and `name`. See `introduction` for details about
         locating elements.
+        
+        Sample:
+        | ${labels1} = | Get List Items | xpath=//h1 |
+        | ${labels2} = | Get List Items | xpath=//h1 | label=${True} |
+        | ${values} = | Get List Items | xpath=//h1 | label=${False} |
+        | Should Be Equal | ${labels1} | ${labels2} |
         """
         select, options = self._get_select_list_options(locator)
-        return self._get_labels_for_options(options)
+        if label:
```

Comment:
I think you did miss understood. Instead of using `label=True` argument (in line 11) we would like to use instead `value=False` argument. And then lines 25 - 26 would look like:
```
if value:
    return self._get_values_for_options(options)
else:
    return self._get_labels_for_options(options)
```
And this would solve the problem explained by @pekkaklarck. I would like to keywords work in same manner that users would not have think that in keyword X string False is actually true and in keyword Y string False is actually false. It could lead in confusing situations for the user point of view.

As for now, it is OK like this:
```
| Get List Items | xpath=//table | value=${FALSE} | # Returns labels |
| Get List Items | xpath=//table | value=FALSE | # Returns values |
```